Assets/VRTK/Scripts/Interactions/VRTK_InteractableObject.cs

```diff
@@ -135,6 +135,22 @@ public enum ValidDropTypes
         /// Emitted when the other object stops using the current object.
         /// </summary>
         public event InteractableObjectEventHandler InteractableObjectUnused;
+        /// <summary>
+        /// Emitted when the object enters a snap drop zone.
+        /// </summary>
+        public event InteractableObjectEventHandler InteractableObjectEnteredSnap;
```

Comment:
I'd keep these similar in name to how  they're named in the snap drop zone.

E.g. SnapDropZone -> `ObjectEnteredSnapDropZone`

So i'd call this `InteractableObjectEnteredSnapDropZone`